ofl/arapey/METADATA.pb

```diff
@@ -24,3 +24,4 @@ fonts {
 subsets: "menu"
 subsets: "latin"
 subsets: "latin-ext"
+minisite_url: "https://arapey.xyz"
```

Comment:
This site is for the new opsz upgrade which hasn't started to be onboarded, so I suggest to remove this here and keep it on an internal tracker of upcoming minisites.